src/main/java/info/freelibrary/iiif/presentation/utils/I18nUtils.java

```
@@ -0,0 +1,121 @@
+
+package info.freelibrary.iiif.presentation.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import info.freelibrary.iiif.presentation.properties.I18n;
+import info.freelibrary.util.Logger;
+import info.freelibrary.util.LoggerFactory;
+
+/**
+ * A utilities class for internationalizations.
+ */
+public final class I18nUtils {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(I18nUtils.class, Constants.BUNDLE_NAME);
+
+    // A naive HTML tag pattern, but something to get us started... for full support requirements, check the spec:
+    // https://iiif.io/api/presentation/3.0/#45-html-markup-in-property-values (should be implemented in IIIF-730)
+    private static final Pattern PATTERN = Pattern.compile("<[a-zA-Z0-9\\-]+>.*<|<[a-zA-Z0-9\\-]+/>", Pattern.DOTALL);
+
+    private I18nUtils() {
+    }
+
+    /**
+     * Determines whether the supplied string has HTML markup in it.
+     *
+     * @param aString A string to check for HTML markup
+     * @return True if the supplied string contains HTML markup; else, false
+     */
+    public static boolean hasHTML(final String aString) {
+        return PATTERN.matcher(aString).find();
+    }
+
+    /**
+     * Strips HTML tags from the strings in the supplied internationalizations.
+     *
+     * @param aI18nsArray An array of internationalizations
+     * @return An array of internationalizations guaranteed not to contain HTML markup
+     */
+    public static I18n[] stripHTML(final I18n... aI18nsArray) {
+        // This is just a placeholder for this for now
+        return aI18nsArray;
```

I know there's a comment that says it's not implemented but maybe it would be better to throw `UnsupportedOperationException` so we don't forget?